scte/Scte35/TimeSignal.py

```diff
@@ -28,6 +28,12 @@ def serialize(self):
     def __str__(self):
         return str(self.splice_time)
 
+    def __iter__(self): #overridding this to return tuples of (key,value)
```

Comment:
Is this the magic that makes line 155 of SpliceEvent work correctly?
`the_dict["time_signal"] = dict(the_dict["time_signal"])`

Reply:
Correct. Dicts are actually considered iterable tuples, hence the Double-under iter lets you do dict() casts.